subprojects/iconv/meson.build

```diff
@@ -17,3 +17,5 @@ iconv_incs = include_directories(
 libiconv = static_library('iconv', iconv_src, include_directories: iconv_incs)
 libiconv_dep = declare_dependency(link_with: libiconv,
                                   include_directories: iconv_incs)
+
+meson.override_dependency('iconv', libiconv_dep)
```

Comment:
Why can't use system iconv? What is this needed for?

Reply:
I believe this is only run if the iconv subproject is explicitly initialized using `subproject()`, which only happens if system iconv is not available: https://github.com/Myaamori/Aegisub/blob/meson-vs2019/meson.build#L61

It's to ensure that the `dependency('iconv')` call in boost_locale resolves to the subproject.

Reply:
Gotcha.